site/en/tutorials/customization/basics.ipynb

```diff
@@ -298,7 +298,7 @@
       "source": [
         "## Datasets\n",
         "\n",
-        "This section uses the `tf.data.Dataset` API to build a pipeline for feeding data to your model. `tf.data.Dataset` is used to build performant, complex input pipelines from simple, re-usable pieces that will feed your model's training or evaluation loops. (Refer to the [tf.data: Build TensorFlow input pipelines](../../guide/data.ipynb) guide to learn more.)"
+        "This section uses the `tf.data.Dataset` API to build a pipeline for feeding data to your model. `tf.data.Dataset` is used to build high-performing, complex input pipelines from simple, re-usable pieces that will feed your model's training or evaluation loops. (Refer to the [tf.data: Build TensorFlow input pipelines](../../guide/data.ipynb) guide to learn more.)"
```

Comment:
"performant" is commonly used - see https://www.tensorflow.org/s/results?q=performant We can keep it. Thank you for the helping improve the docs.